starknet_py/contract.py

```diff
@@ -235,7 +235,6 @@ def __init__(
         payload_transformer: FunctionSerializationAdapter,
         contract_data: ContractData,
         max_fee: Optional[int],
-        version: int,
```

Comment:
Let's include this removal in the `migration_guide.rst`. 

There's a guide on building building docs locally here if you need it https://starknetpy.readthedocs.io/en/latest/development.html#documentation